controllers/actions.github.com/autoscalinglistener_controller_test.go

```diff
@@ -15,7 +15,7 @@ import (
 	. "github.com/onsi/gomega"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
-	actionsv1alpha1 "github.com/actions/actions-runner-controller/apis/actions.github.com/v1alpha1"
```

Comment:
I would avoid changing this in this PR to reduce the amount of PR diff. 😄

Reply:
Yup, reverting 👍

Reply:
Diff only got 7 lines smaller 😭